testdata/xml-data/test-data-systematic.xml

```diff
@@ -281,6 +312,22 @@
         </text-prop>
     </resource>
 
+    <resource label="iiif-link with license / copyright"
+              restype=":ImageThing"
+              id="iiif-link-license-copyright"
+              permissions="open">
+        <iiif-uri
+                license="CC-BY-NC"
+                copyright="copyright_id"
+        >https://iiif.wellcomecollection.org/image/b20432033_B0008608.JP2/full/1338%2C/0/default.jpg</iiif-uri>
```

Comment:
```suggestion
        <iiif-uri license="CC-BY-NC" copyright="copyright_id">
            https://iiif.wellcomecollection.org/image/b20432033_B0008608.JP2/full/1338%2C/0/default.jpg
        </iiif-uri>
```

Reply:
This is not possible, because that makes the URI a multi-line text, which is prohibited.